photutils/datasets/tests/test_make.py

```diff
@@ -22,6 +27,13 @@
 TABLE['y_stddev'] = [2, 1, 3.5]
 TABLE['theta'] = np.array([0., 30, 50]) * np.pi / 180.
 
+TABLE2 = Table()
```

Comment:
What about `TABLE_PRF` instead of `TABLE2`?  While you're at it I suppose you could change them both to something more like `SOURCES_TABLE` since "table" is pretty ambiguous.

Reply:
I have fixed the table names to be better. I liked ``SOURCES_TABLE`` so we now have ``SOURCES_TABLE`` and ``SOURCES_TABLE_PRF``.